corgi/tasks/brew.py

```diff
@@ -29,17 +29,21 @@ def slow_fetch_brew_build(build_id: int, save_product: bool = True, force_proces
     else:
         logger.info("Fetching brew build with build_id: %s", build_id)
 
-    build = Brew().get_component_data(build_id)
-    build_meta = build["build_meta"]["build_info"]
+    component = Brew().get_component_data(build_id)
+    if not component:
+        logger.info("No data fetched for build %s from Brew, exiting...", build_id)
+        return
```

Comment:
I'd rather make this an uncaught exception, so we can see how may builds of an unsupported type we keep trying to ingest. Or at least error-level logging if we want to silence this, so it's easier to find in logs.

Reply:
Ah, I see now. I was looking at an old version of the diff earlier. Error logging sounds fine then.